backend/app/api/endpoints/webhooks/callback.py

```
@@ -0,0 +1,137 @@
+
+    return {
+        "status": "processed",
+        "subtask_id": subtask_id,
+        "resumed": result.get("resumed", False),
+    }
+
+
+@router.get("/callback/token/{subtask_id}")
+async def get_callback_url(
+    subtask_id: int,
+    db: Session = Depends(get_db),
```

Ruff (0.14.6)

Unused function argument: `db`

(ARG001)